netmap/netmap_contract.go

```diff
@@ -73,7 +73,8 @@ const (
 	containerContractKey = "containerScriptHash"
 	balanceContractKey   = "balanceScriptHash"
 
-	cleanupEpochMethod = "newEpoch"
+	newEpochSubscribersKey = "newEpochSubscribers"
```

Comment:
```suggestion
	newEpochSubscribersKeyPrefix = "newEpochSubscribers"
```

Reply:
Please use one byte prefixes, they don't have to be this long and we can only have 64-byte keys.

Reply:
> newEpochSubscribersKeyPrefix

ok but without `key` then

> Please use one byte prefixes

`n` won my democratic election